packages/kbn-types/.gitignore

```diff
@@ -0,0 +1 @@
+types/
```

Comment:
Moved it to `@elastic`. We'll likely never push anything that's not under `@elastic`, so don't really see the point of that folder, but it's something we can solve later